apps/showcase/src/style/dark-theme/dark-theme.scss

```diff
@@ -6,11 +6,6 @@
 --bs-body-color: #ffffff;
```

Comment:
should be regenerated with `--df` instead of `--bs`

Reply:
... I noticed I remove accidentally the file in the previous update (it has been recreated by the design token process) so I reseted it but I forgot to rerun the design token :'(